### DISCUSSION TITLE
Unify discussion channels

### DISCUSSION THREAD

**Discussion by kordano** [replikativ/datahike#495] [Ideas]
Currently, we have too many channels for interacting with the community and sometimes it's difficult to keep up.
I propose that we would have github for 
- bug reports
- feature requests
- discussions that we would like to preserve

For direct messages I would favor slack over Discord and close the latter one. And if there are longer discussions on slack I would direct them here instead.

**Comment by TimoKramer** [comment]
We could have a public meeting open for the community on slack instead of Discord. Since it is a paid version for now slack has the feature called 'huddle' that we can use for this.

**Comment by TimoKramer** [comment]
We should remove the Discord-Link from the website datahike.io

**Reply by TimoKramer** [reply]
and from the README
